quint/apalache-tests.md

```diff
@@ -35,51 +35,14 @@ error: Configuration error (see the manual): Operator invalidInit not found (use
 
 ## Successful verification
 
-### Can verify `../examples/language-features/booleans.qnt`
-
-<!-- !test check can check booleans.qnt -->
-```
-quint verify --init Init --step Next ../examples/language-features/booleans.qnt
```

Comment:
Why are we removing these?

afaict the examples script runs the commands in a subshell. Does it fail if one of the commands fail?

Reply:
It will fail in CI if the outcome is different then we expect.

Reply:
I’m open to rethinking the approach here ofc. If you foresee problems.

but, as noted in my PRs, my current thought is that we move tests that the examples Work or not per se into the examples dashboard, and only use the integration tests for checking specific functionality.